applications/tari_base_node/src/builder.rs

```diff
@@ -507,6 +507,7 @@ where
         factories.clone(),
         config.base_node_query_timeout,
         config.transaction_base_node_monitoring_timeout,
+        config.transaction_chain_monitoring_timeout,
```

Comment:
This sounds very similar to the setting above. Perhaps a better name is `transaction_mined_monitoring_timeout`?